src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -741,7 +741,7 @@ impl<Tag: Copy, Extra> Allocation<Tag, Extra> {
 type Block = u64;
 
 /// A bitmask where each bit refers to the byte with the same index. If the bit is `true`, the byte
-/// is defined. If it is `false` the byte is undefined.
+/// is initialized. If it is `false` the byte is uninitialized.
 #[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, RustcEncodable, RustcDecodable)]
 #[derive(HashStable)]
 pub struct UninitMask {
```

Comment:
So given the comment above this, I feel like `InitMask` would actually be a better term? (Cc @oli-obk) That should also make it more clear that in `set_range`, `true` means "initialized". (The field name and getter should then change to `init_mask`, accordingly.)

Sorry for the back-and-forth. Naming things right is a hard problem.

Reply:
Heh, yea, that makes sense. The double or triple negatives around uninit mask have confused me a few times

Reply:
@hbina so to avoid renaming twice, could you do that in this PR?